data/src/main/java/com/architect/coders/mu8/data/utils/DataExtensions.kt

```diff
@@ -8,5 +11,4 @@ const val MARVEL_PRIVATE_KEY = BuildConfig.MARVEL_PRIVATE_KEY
  * Create hash required to any request to Marvel API
  * @return md5(ts+privateKey+publicKey)
  */
-fun createHashCode(timeStamp: String = TIME_STAMP) =
-    (timeStamp + MARVEL_PRIVATE_KEY + MARVEL_PUBLIC_KEY).md5()
\ No newline at end of file
+fun createHashCode() = (TIME_STAMP + MARVEL_PRIVATE_KEY + MARVEL_PUBLIC_KEY).md5()
```

Comment:
Use `${}` for the constants intead of the `+` concatenation`

Reply:
done